plugins/bulk-import/src/components/AddRepositories/AddRepositoriesForm.test.tsx

```diff
@@ -97,7 +109,9 @@ describe('AddRepsositoriesForm', () => {
             ],
           ]}
         >
-          <AddRepositoriesForm error={null} />
+          <QueryClientProvider client={queryClient}>
+            <AddRepositories error={null} />
```

Comment:
I see that it's already before like this, but can you make the `error` prop optional and remove the prop here?